docs/src/pages/components/authenticator/index.page.mdx

```diff
@@ -373,13 +373,14 @@ The following example customizes these slots with:
 
 The `Authenticator` ships with [translations](https://github.com/aws-amplify/amplify-ui/blob/main/packages/ui/src/i18n/translations.ts) for:
 
+- `de` – German
 - `en` – English (default)
-- `zh` – Chinese
+- `es` – Spanish
 - `fr` – French
-- `de` – German
 - `it` – Italian
 - `ja` – Japenese
-- `es` – Spanish
+- `pl` - Polish
+- `zh` – Chinese
```

Comment:
Ah, my bad, I meant alphabetical with the language names to the right (with English at the top). I'll readjust it, sorry for poorly explaining it!